src/main/java/com/google/sps/servlets/MatchServlet.java

```diff
@@ -49,8 +73,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //return null for matches so that page redirects to logged out homepage
         response.getWriter().println(gson.toJson(null));
     } else {
-        Collection<User> matches = matchRepository.getMatchesForUser(sessionContext.getLoggedInUser());
-        response.getWriter().println(gson.toJson(matches));
+        boolean isRequestPassed = analyzeTimeOfRequest(request);
+        // if true, means the request was passed and added to the backend to get processed
```

Comment:
What does the request passing mean?

Reply:
Request passing means it was accepted into the backend, and not blocked